### DISCUSSION TITLE
AttributeError: module 'faiss' has no attribute 'StandardGpuResources'

### DISCUSSION THREAD

**Discussion by Geeker-LStar** [facebookresearch/faiss#3706] [Q&A]
Dear developer:
I used faiss-gpu version 1.7.4 on my Win11 system. Here are version info:

> Name: faiss
> Version: 1.7.4
> Summary: A library for efficient similarity search and clustering of dense vectors
> Home-page: https://github.com/facebookresearch/faiss
> Author: Matthijs Douze, Jeff Johnson, Herve Jegou, Lucas Hosseini
> Author-email: matthijs@fb.com
> License: MIT
> Location: D:\Anaconda\Lib\site-packages
> Requires: numpy
> Required-by:

However I received an error:

> AttributeError: module 'faiss' has no attribute 'StandardGpuResources'

How can I deal with it??

THANK YOU VERY MUCH!!!

**Comment by kuarora** [comment]
Is this an error at compile time? Or is this an error on runtime?
Could you provide more detail?

**Comment by mdouze** [comment]
This is because Faiss is not compiled with GPU support.

**Reply by junjieqi** [reply]
@mdouze would you like to provide more context regarding with it? The user installed it from faiss-gpu and I assume it should be compiled with GPU. Or I missed some parts.

**Reply by mdouze** [reply]
We don't support faiss-gpu on windows, but there is a supported package on conda-forge IIUC.
